core_planning/mpc_follower/launch/mpc_follower.launch

```diff
@@ -37,18 +39,29 @@
 
 	<group ns="$(arg ns)">
 
-		<node pkg="mpc_follower" type="mpc_waypoints_converter" name="mpc_waypoints_converter" output="screen"/>
+		<node pkg="mpc_follower" type="mpc_waypoints_converter" name="mpc_waypoints_converter" output="screen">
+			<remap from="/base_waypoints" to="/guidance/base_waypoints"/>
+			<remap from="/closest_waypoint" to="/guidance/closest_waypoint"/>
+			<remap from="/final_waypoints" to="/guidance/carma_final_waypoints"/>
```

Comment:
Don't remap the global namespace like this as it becomes hard to track down. Better to remove the ns argument in this group tag and then just remap the base names.